BlueHarvest/FailClient.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  FailClient.swift
+//  BlueHarvest
+//
+//  Created by David Nutter on 3/20/17.
+//  Copyright © 2017 David Nutter. All rights reserved.
+//
+
+import Foundation
+
+struct FailClient: APIClient {
```

Comment:
Yes I was using these clients during the development process. During `ViewModel` testing  I was swapping in each of the clients until finally using `APIClient`. These would definitely be included in a test suite, along with the ProfileFormatter